docs/crds/gslbhostrule.md

```diff
@@ -16,6 +16,7 @@ spec:
   thirdPartyMembers:
   - site: non-avi-site
     vip: 10.10.10.10
+    publicIP: 122.162.150.96
```

Comment:
are we supporting ipv6 here? If yes, have we tested it?

Reply:
have added IPv4 address in description since here also testing with IPv6 is not complete